public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.ts

```diff
@@ -237,7 +238,14 @@ export function gridItemToGridLayoutItemKind(gridItem: DashboardGridItem, isSnap
 
 function getElements(state: DashboardSceneState) {
   const panels = state.body.getVizPanels() ?? [];
-  const panelsArray = panels.reduce((acc: PanelKind[], vizPanel: VizPanel) => {
+  const panelsArray = panels.map((vizPanel: VizPanel) => {
+    const vizFieldConfig: FieldConfigSource = {
+      ...vizPanel.state.fieldConfig,
+      defaults: {
+        ...vizPanel.state.fieldConfig,
```

Comment:
should these both be spreading the same thing?

Reply:
@joshhunt  - think this is addressed already, aren't u looking at cached version?

Reply:
yeah, i think i went to source from an old commit link :(